examples/employee-app/src/main/java/io/helidon/service/employee/EmployeeRepositoryImpl.java

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2018, 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.service.employee;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.UUID;
+
+/* EmployeeRepositoryImpl
+    Note: The strings id, name, and other search strings are validated before 
+    being passed to the methods in this class.
+*/
+
+public class EmployeeRepositoryImpl implements EmployeeRepository {
+
+    private static final CopyOnWriteArrayList<Employee> eList = EmployeeMockList.getInstance();
```

Comment:
I would suggest getting rid of the EmployeeMockList class, and instead load the list of employee in the constructor using a json file loaded from classloader resource and jsonb (i.e take the suggestion I made in the comment in EmployeeMockList and move the code in the constructor of this class).

Reply:
Done.

Reply:
I see you've made the changes in EmployeeMockList, however the class remain, now all you need to do is move the code in the constructor of this class (EmployeeRepositoryImpl)

Reply:
Done